pyServer/AzureDiskInspectService.py

```diff
@@ -262,7 +262,12 @@ def ParseUrlArguments(self, urlPath, sasKey):
             container_blob_name = container_blob_name + '/' + urlSplit[urlSplitIndex]
             urlSplitIndex = urlSplitIndex + 1
         
-        storageUrl = urllib.parse.urlunparse(
+        self.telemetryLogger.info("Storage Account from requestUri: " + storageAcctName)
+        if ".blob." in storageAcctName:
```

Comment:
This looks good. Consider adding a small comment here on the two different storage account parsing logic.

Reply:
Comment added